tools/wheel_resolver/__init__.py

```diff
@@ -85,39 +84,39 @@ def main(
     PyPI for PACKAGE with VERSION.
 
     """
-    for u in url:
-        response = requests.head(u)
-        if response.status_code != requests.codes.ok:
-            _LOGGER.warning(
-                "%s-%s is not available, tried %r", package_name, package_version, u
-            )
-        else:
-            click.echo(u)
-            return
+    try:
+        output_name = output.get()
+    except output.OutputNotSetError:
+        _LOGGER.error("could not get $OUTS")
+        sys.exit(1)
+
+    if output.download(package_name, package_version, url, output_name):
+        return
 
-    # We're currently hardcoding PyPI but we should consider allowing other
-    # repositories
-    # TODO (tm-jdelapuente): allow downloads from other package repositories
```

Comment:
Why removing this TODO, if not done? I wonder whether a flag could be added with a default?

Reply:
IMO we're (to a great extent) satisfying this requirement because we're now handling downloads if a URL is passed.

What we don't support is resolving the wheel to download from multiple options in a single repo. Maybe we can create an issue for that if someone feels strongly about it, otherwise this feels like a wishlist item to me